.gitignore

```diff
@@ -7,6 +7,8 @@ node_modules
 .DS_STORE
 coverage
 dist
+.output
+
 lib/cypress/videos
 lib/cypress/screenshots
-.output
\ No newline at end of file
+lib/README.md
```

Comment:
Why ignore the README.md if it's deleted?

Reply:
Because it's being generated each time we build the SDK locally to avoid future commits